samples/subsys/usb/hid/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _usb_hid:
+
+USB HID Sample Application
+##########################
+
+Overview
+********
+
+This sample app demonstrates use of a USB Human Interface Device (HID) driver
+by the Zephyr project.  This very simple driver enumerated as a raw HID device.
+This sample can be found under :file:`samples/subsys/usb/hid` in the
+Zephyr project tree.
+
+Requirements
+************
+
+This project requires an USB device driver.
+
+Building and Running
+********************
+
+This sample can be built for multiple boards, in this example we will build it
+for the quark_se_c1000_devboard board:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/hid
+   :board: quark_se_c1000_devboard
+   :goals: build
+   :compact:
+
+After you have built and flashed the sample app image to your board, plug the
+board into a host device, for example, a PC running Linux.
+The board will be detected as shown by the Linux dmesg command:
```

Comment:
You wrote you were using the ``dmesg`` command, but you used ``journalctl`` in your example below.